src/Microsoft.PowerApps.TestEngine/Modules/TestEngineExtensionChecker.cs

```diff
@@ -385,6 +445,111 @@ public bool VerifyContainsValidNamespacePowerFxFunctions(TestSettingExtensions s
             return isValid;
         }
 
+        // Helper method to convert wildcard patterns to regular expressions
+        private string WildcardToRegex(string pattern)
+        {
+            return "^" + Regex.Escape(pattern).Replace("\\*", ".*").Replace("\\?", ".") + "$";
+        }
+
+        private bool CheckPropertyArrayContainsValue(TypeDefinition typeDefinition, string propertyName, out string[] values)
+        {
+            // Find the property by name
+            var property = typeDefinition.Properties.FirstOrDefault(p => p.Name == propertyName);
+            if (property == null)
+            {
+                values = null;
```

Comment:
you can initialize "values = null;" at the beginning of the function and avoid all the rest of these assignments across of it.

Reply:
Updated